packages/data-driver-postgres/src/query/index.ts

```diff
@@ -18,16 +20,18 @@ import { orderBy } from './orderBy.js';
  * @param query - The abstract SQL statement
  * @returns An actual SQL query with parameters
  */
-export function constructSqlQuery(query: AbstractSqlQuery): ParameterizedSQLStatement {
-	const statementParts = [select, from, where, orderBy, limit, offset];
+export function constructSqlQuery(query: AbstractSqlQuery): ParameterizedSqlStatement {
```

Comment:
since we always pass the whole query to the query part functions like `select`, `where`, etc. here, the test files got a little bigger than needed since the whole query object needs to be "mocked" instead of the nodes in question only. I would do it differently next time since it was a bit of unnecessary work, but not sure if it's worth it change it now afterwards.. if you have an opinion on this, let me know